pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>su.litvak.chromecast</groupId>
     <artifactId>api-v2</artifactId>
     <version>0.9.2-SNAPSHOT</version>
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```

Comment:
My main concern is that I don't know what the 'bundle' means here and it's not a standard packaging time for maven project (see http://maven.apache.org/pom.html#Maven_Coordinates). It confuses me and I don't want to spend time investigating why it is put there and what it will mean. Neither I want the library users to spend their time for doing same investigation. I believe that this is a quite more specific thing than the 'jar' packaging. I think that this parameter is one of the major parameter of the maven projects.

 So, if you want it to be merged then please change it to the scheme where it updates the manifest as said in that link.